src/data_loading.jl

```diff
@@ -74,4 +86,14 @@ function load(::MetabolicProfiles; timepoint_metadata = load(Metadata()))
     comm = CommunityProfile(Matrix(df[!, r"^FE"]), ms, MicrobiomeSample.(names(df, r"^FE")))
     set!(comm, timepoint_metadata; namecol=:etoh)
     return comm
+end
+
+function load(::Neuroimaging)
+    df = CSV.read(datafiles("exports", "raw_brain_measures.csv")) ## Have the raw_brain_measures.csv file in your exports directory.
+    subjs, tpts = split_subject_timepoint(df."Measure:volume")
+        df."Left-Thalamus" = df."Left-Thalamus" .+ df."Left-Thalamus-Proper"
```

Comment:
Can we just add a couple of gut-checks here? Something like 

```
all(row-> row."Left-Thalamus" == 0 ⊻ row."Left-Thalamus-Proper", eachrow(df)) || error()
```

for each pair.

Reply:
Also, why is this line indented?

Reply:
```suggestion
        df."Left-Thalamus"  .+= df."Left-Thalamus-Proper"
```